Added topic

Storylover-Vodhr opened the pull request:
Topic found between 3154 and 3169

Storylover-Vodhr commented:
Hmmm...

I have no idea why it's not passing it's test. Can someone possibly shed some light onto my failures?

multimokia commented:
Looking into it

Legendkiller21 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,25 @@ label monika_ribbon:
+    m 1eub "On the bright side..."
+    m "Once I cross over, if you remember to bring the sunscreen..."
+    m 1tfu "I might need help putting it on."
+    m 1hub "Ahahahaha!
```
```suggestion
    m 1hub "Ahahahaha!"
```

multimokia commented:
Ah, I see it. You didn't close the quotation on line 3166

Legendkiller21 commented:
I was 15 seconds too late.

Storylover-Vodhr commented:
I spent 20 minutes looking at my Opening lines, assuming that I was just both blind and a moron. -_-

Thank you two so much.

jmwall24 commented:
So I'm not the only one who thought about camping lol #1719

Rai99 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,25 @@ label monika_ribbon:
+    m 1tfu "I might need help putting it on."
+    m 1hub "Ahahahaha!"
+    m 1efu "I'm only teasing you, [player]."
+    m 1tfu "Well, at least a little."
```
I'd add an "ehehe" to end with.

Storylover-Vodhr replied:
Okie dokie. Those are all easy tweaks, and I'll see if I can expand it a bit, as well as reference the backpacking option.

And, I got several outdoors-esque ideas, but I can change the category.

Storylover-Vodhr replied:
Drat. Looked at the backpacking one; It's great, but it possibly cuts out the lake idea I had. XP

Storylover-Vodhr replied:
Wait, I might be able to make it work.

Rai99 commented:
Unfortunately, I think this topic is too much of an overlap of that referenced "Backpacking" topic. Otherwise, I think it could have been a nice addition.

Storylover-Vodhr commented:
> Unfortunately, I think this topic is too much of an overlap of that referenced "Backpacking" topic. Otherwise, I think it could have been a nice addition.
> 
> It may be possible to have this work with the other topic however.

I changed it a bit, and added the second, sister topic I was almost finished with. Hopefully it's not trash.
(Although, I admit some difficulties with proper expressions.

Storylover-Vodhr commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,51 @@ label monika_ribbon:
+    m 1sub "Of how one would hike up through the forests and trees."
+    m 1eub "Climbing up cliff faces and trudge through streams."
+    m "Hearing nothing but the birds and the sounds of the mountain as you made your way up it's heights."
+    m 5rub "And finally... after all your effort and struggles..."
```
And what would a dissolve be? XD

Rai99 replied:
show monika ____ at t11 zorder MAS_MONIKA_Z with dissolve
m ____ ""

Storylover-Vodhr commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,51 @@ label monika_ribbon:
+    m "Hearing nothing but the birds and the sounds of the mountain as you made your way up it's heights."
+    m 5rub "And finally... after all your effort and struggles..."
+    m 5eub "Finding yourself standing at the top, knowing that you made it, seeing the testiment to your success around you."
+    m "I...I truly want to share that with you."
```
Alrighty! In all honesty, I want to add the choice option to the mountain topic, but I haven't the slightest idea how. Is it basically just a 

show monika 1eua
    menu:
        m "Yes or no?"
        "Yes.":
            $ persistent._mas_pm_yes = True
            m 3eub "That's really cool!"
           

        "No.":
            $ persistent._mas_pm_yes = False
            m 3eub "Oh. Ok!"

Storylover-Vodhr replied:
Or is it something else?

Rai99 replied:
Yeah, it's like that.

Storylover-Vodhr replied:
Alrighty! I shall make a seperate path, then.

Storylover-Vodhr replied:
Aaaand, done. It's a bit weak though.

multimokia commented:
You don't need the transform (at t11) by the way.

Storylover-Vodhr commented:
> You don't need the transform (at t11) by the way.

Oh? 

Which one? The second one, after the no, or the first one?

multimokia commented:
Also, you're failing Travis because you're using `tab` characters. To indent, you'll want to line up using spaces instead of hitting Tab.

Storylover-Vodhr commented:
Welp, I give up. My limit before asking for help is around 15 attempts.

multimokia commented:
Never mind about the transform bit. You do need it. That bit is fine.

Storylover-Vodhr commented:
> Also, you're failing Travis because you're using `tab` characters. To indent, you'll want to line up using spaces instead of hitting Tab.

Removed the tabs. Anything else I'm borking? XD

multimokia commented:
By the way, you can check the log that Travis spits out, it tells you why it failed

Storylover-Vodhr commented:
Thank you.

Storylover-Vodhr commented:
I apologize if I'm spamming the feed.

multimokia commented:
Still got more tab characters, looks like

Storylover-Vodhr commented:
> Still got more tab characters, looks like

Nah, I got them. Thank you for the heads up about the checker. I was checking it the old fashion way, looking over other's code and comparing it to the stuff I was making.

Storylover-Vodhr commented:
Oh, thank god.

multimokia commented:
Ey

Rai99 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,60 @@ label monika_ribbon:
+            m 5eubla "Well...I hope that one day, we'll have that chance. To make it to the top of our mountain."
+            m 5hua "And I'll do anything to give us that shot."
+            
+        "No.":
```
I think this "No" option here is a bit flat and could be expanded on further.

Rai99 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,60 @@ label monika_ribbon:
+
+label monika_outdoors:
+    m 1eua "Do you ever go camping, [player]?"
+    m 3eua "It's a wonderful way to relax and get some fresh air,"
```
I think this and the next line should be on one line.

Rai99 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,60 @@ label monika_ribbon:
+    m "As well as to see the areas around you, to visit state and national parks."
+    m 1huu "It's almost like a more relaxed backpacking trip!"
+    m 1eua "But, while it is a good way to spend time outdoors, there are several dangers that most people don't bother to think about."
+    m 3lud "Such as bug spray or sunscreen. Many people forget or even forgo them; thinking them unimportant."
```
I don't think this sentence should start with "Such".

Rai99 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,60 @@ label monika_ribbon:
+    m 1huu "It's almost like a more relaxed backpacking trip!"
+    m 1eua "But, while it is a good way to spend time outdoors, there are several dangers that most people don't bother to think about."
+    m 3lud "Such as bug spray or sunscreen. Many people forget or even forgo them; thinking them unimportant."
+    m 1eud "But without it, sunburns are almost inevitable, and many insects carry diseases that can really harm you."
```
I'd get rid of "But" and just start with "Without it,".

Rai99 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,60 @@ label monika_ribbon:
+    m 1tfu "I might need help putting it on."
+    m 1hub "Ahahaha!"
+    m 1efu "I'm only teasing you, [player]."
+    m 1tfu "Well, at least a little."
```
I really think this should be a smug wink, but the only wink is 1kua which isn't smug. 

Maybe we could add some more in.

Rai99 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,60 @@ label monika_ribbon:
+    m 1eua "Have you ever been in the mountains, [player]?"
+    m "And I don't mean driving through them, or in a mountain town."
+    m "I mean really up there. In the fresh air, tens of thousands of feet up, seeing the rest of world below you."
+    m "..."
```
I think this line could use a thinking expression.

Rai99 commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,60 @@ label monika_ribbon:
+
+label monika_mountain:
+    m 1eua "Have you ever been in the mountains, [player]?"
+    m "And I don't mean driving through them, or in a mountain town."
```
I'd get rid of "And".

Storylover-Vodhr replied:
Changed most, with the exception of the thinking expression. Not because I don't want to, but because I can't seem to find a fitting one.

Storylover-Vodhr replied:
Well, I must go, I gotta get some sleep. I'll read all con crit once I return, as well as make a couple more topics. 

Now that I know where I keep screwing up, I should be vastly more efficient.

ThePotatoGuy commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,59 @@ label monika_ribbon:
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_outdoors",category=['misc'],prompt="Camping safety",random=True))
```
add `nature` as a category. 
```suggestion
    addEvent(Event(persistent.event_database,eventlabel="monika_outdoors",category=['misc', "nature"],prompt="Camping safety",random=True))
```

ThePotatoGuy commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3147,6 +3147,59 @@ label monika_ribbon:
+    return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_mountain",category=['misc'],prompt="Mountains",random=True))
```
add `nature` as a category
```suggestion
    addEvent(Event(persistent.event_database,eventlabel="monika_mountain",category=['misc', "nature"],prompt="Mountains",random=True))
```

Storylover-Vodhr replied:
Added Nature as a category, as requested.

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+
+label monika_outdoors:
+    m 1eua "Do you ever go camping, [player]? It's a wonderful way to relax and get some fresh air,"
+    m 3eua "As well as to see the areas around you, to visit state and national parks."
```
```suggestion
    m 3eub "It's a wonderful way to relax and get some fresh air and see the parks around you!"
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+label monika_outdoors:
+    m 1eua "Do you ever go camping, [player]? It's a wonderful way to relax and get some fresh air,"
+    m 3eua "As well as to see the areas around you, to visit state and national parks."
+    m 1huu "It's almost like a more relaxed backpacking trip!"
```
```suggestion
    m 1huu "It's almost like a more relaxed backpacking trip actually!"
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1eua "Do you ever go camping, [player]? It's a wonderful way to relax and get some fresh air,"
+    m 3eua "As well as to see the areas around you, to visit state and national parks."
+    m 1huu "It's almost like a more relaxed backpacking trip!"
+    m 1eua "But, while it is a good way to spend time outdoors, there are several dangers that most people don't bother to think about."
```
```suggestion
    m 1eka "But while it is a good way to spend time outdoors, there are several dangers that most people don't bother to think about."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 3eua "As well as to see the areas around you, to visit state and national parks."
+    m 1huu "It's almost like a more relaxed backpacking trip!"
+    m 1eua "But, while it is a good way to spend time outdoors, there are several dangers that most people don't bother to think about."
+    m 3lud "A good example would be bug spray or sunscreen. Many people forget or even forgo them; thinking them unimportant."
```
```suggestion
    m 3eud "A good example would be bug spray or sunscreen. Many people forget or even forgo them;{w=0.5} thinking they're unimportant..."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1huu "It's almost like a more relaxed backpacking trip!"
+    m 1eua "But, while it is a good way to spend time outdoors, there are several dangers that most people don't bother to think about."
+    m 3lud "A good example would be bug spray or sunscreen. Many people forget or even forgo them; thinking them unimportant."
+    m 1eud "Without it, sunburns are almost inevitable, and many insects carry diseases that can really harm you."
```
```suggestion
    m 1eksdld "and without them, sunburns are almost inevitable, and many insects carry diseases that can really harm you."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 3lud "A good example would be bug spray or sunscreen. Many people forget or even forgo them; thinking them unimportant."
+    m 1eud "Without it, sunburns are almost inevitable, and many insects carry diseases that can really harm you."
+    m "So, please promise me that the next time you go outdoors, be it camping or backpacking, you won't forget them."
+    m "It may be a bit of a pain, but if you don't use them, you might end up miserable, or even get really sick."
```
```suggestion
    m "While it may be a bit of a pain, if you don't use them...{w=0.5}you might end up miserable or even get really sick."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1eud "Without it, sunburns are almost inevitable, and many insects carry diseases that can really harm you."
+    m "So, please promise me that the next time you go outdoors, be it camping or backpacking, you won't forget them."
+    m "It may be a bit of a pain, but if you don't use them, you might end up miserable, or even get really sick."
+    m 1eub "But, on the bright side..."
```
All of this here should be in an if statement for condition `mas_isMoniAff(higher=True)` I'm thinking.

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1eua "But, while it is a good way to spend time outdoors, there are several dangers that most people don't bother to think about."
+    m 3lud "A good example would be bug spray or sunscreen. Many people forget or even forgo them; thinking them unimportant."
+    m 1eud "Without it, sunburns are almost inevitable, and many insects carry diseases that can really harm you."
+    m "So, please promise me that the next time you go outdoors, be it camping or backpacking, you won't forget them."
```
This can go into an `else` block for the previously mentioned condition.

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1eub "But, on the bright side..."
+    m "Once I cross over, if you remember to bring the sunscreen..."
+    m 1tfu "I might need a little help putting it on."
+    m 1hub "Ahahaha!"
```
Too many `ha`'s
```suggestion
    m 1hub "Ahaha!"
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1tfu "I might need a little help putting it on."
+    m 1hub "Ahahaha!"
+    m 1efu "I'm only teasing you, [player]."
+    m 1tfu "Well, at least a little. Ehehe."
```
Minor thing, but I think it's kinda cute.
```suggestion
    m 1tfu "Well, at least a little. Ehehe~"
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+
+label monika_mountain:
+    m 1eua "Have you ever been in the mountains, [player]?"
+    m "I don't mean driving through them, or in a mountain town, either."
```
```suggestion
    m 1rksdla "I don't mean driving through them or in a mountain town..."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+label monika_mountain:
+    m 1eua "Have you ever been in the mountains, [player]?"
+    m "I don't mean driving through them, or in a mountain town, either."
+    m "I mean really up there. In the fresh air, tens of thousands of feet up, seeing the rest of world below you."
```
```suggestion
    m 3hub "I mean really up there. In the fresh air, tens of thousands of feet up, seeing the rest of world below you."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1eua "Have you ever been in the mountains, [player]?"
+    m "I don't mean driving through them, or in a mountain town, either."
+    m "I mean really up there. In the fresh air, tens of thousands of feet up, seeing the rest of world below you."
+    m "..."
```
```suggestion
    m 2dtc "..."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m "I don't mean driving through them, or in a mountain town, either."
+    m "I mean really up there. In the fresh air, tens of thousands of feet up, seeing the rest of world below you."
+    m "..."
+    m 1eub "I've always wanted to try that."
```
I think this line and the next can be merged into one:
```suggestion
    m 3eub "I've always wanted to try that, but I've never had the chance. I've only read about it."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m "I mean really up there. In the fresh air, tens of thousands of feet up, seeing the rest of world below you."
+    m "..."
+    m 1eub "I've always wanted to try that."
+    m "But I've never had the chance. I've only read about it."
```
Then we won't need this one.
```suggestion
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m "..."
+    m 1eub "I've always wanted to try that."
+    m "But I've never had the chance. I've only read about it."
+    m 3eub "But the stories were breathtaking."
```
```suggestion
    m 3wuo "The stories were breathtaking!"
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1eub "I've always wanted to try that."
+    m "But I've never had the chance. I've only read about it."
+    m 3eub "But the stories were breathtaking."
+    m 1sub "Of how one would hike up through the forests and trees."
```
These all kinda sound like extending thoughts off each other, so we can let them continue into each other, I think.
```suggestion
    m 1sub "How one would hike up through the forests and trees..."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m "But I've never had the chance. I've only read about it."
+    m 3eub "But the stories were breathtaking."
+    m 1sub "Of how one would hike up through the forests and trees."
+    m 1eub "Climbing up cliff faces and trudge through streams."
```
```suggestion
    m 1eub "Climbing up cliff faces and trudge through streams..."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 1eub "Climbing up cliff faces and trudge through streams."
+    m "Hearing nothing but the birds and the sounds of the mountain as you made your way up its heights."
+    show monika 5rub at t11 zorder MAS_MONIKA_Z with dissolve 
+    m 5rub "And finally... after all your effort and struggles..."
```
```suggestion
    m 5rub "And finally...after all the effort and struggles..."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    show monika 5rub at t11 zorder MAS_MONIKA_Z with dissolve 
+    m 5rub "And finally... after all your effort and struggles..."
+    m 5eub "Finding yourself standing at the top, knowing that you made it, seeing the testiment to your success around you."
+    m "I... I truly want to share that with you."
```
```suggestion
    m "I...I truly want to share that with you."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+    m 5rub "And finally... after all your effort and struggles..."
+    m 5eub "Finding yourself standing at the top, knowing that you made it, seeing the testiment to your success around you."
+    m "I... I truly want to share that with you."
+    m "To make it to the top of the mountain, and look around at our successes. To see our struggles behind us, and feel proud at what we've done."
```
```suggestion
    m "To make it to the top of the mountain, and look around at our successes. To see our struggles behind us, and feel proud of what we've done."
```

multimokia commented on Monika After Story/game/script-topics.rpy:
```diff
@@ -3434,6 +3434,83 @@ label monika_ribbon:
+        "Yes.":
+            $ persistent._mas_pm_would_like_mt_peak = True
+
+            m 5eubla "Well...I hope that one day, we'll have that chance. To make it to the top of our mountain."
```
```suggestion
            m 5eubla "Well... I hope that one day, we'll have that chance. To make it to the top of our mountain."
```
